tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26, py27, py33, py34, py35
+envlist = py26, py27, py33, py34, py35, py36
```

Comment:
I know, but IMHO code should be compatible with python 3 to better switching.
But if you want I can remove this file.